src/components/core/geoChart/GeoChartLegendRenderer.tsx

```diff
@@ -1,48 +1,43 @@
 // (C) 2020 GoodData Corporation
 import * as React from "react";
 import without = require("lodash/without");
-import { Execution } from "@gooddata/typings";
-import { stringToFloat } from "../../../helpers/utils";
+// import { Execution } from "@gooddata/typings";
+// import { stringToFloat } from "../../../helpers/utils";
 import { IntlWrapper } from "../../core/base/IntlWrapper";
 import { IntlTranslationsProvider, ITranslationsComponentProps } from "../../core/base/TranslationsProvider";
 import PushpinSizeLegend from "./legends/PushpinSizeLegend";
-import { getGeoData, getFormatFromExecutionResponse } from "../../../helpers/geoChart";
+// import { getGeoData, getFormatFromExecutionResponse } from "../../../helpers/geoChart";
 import { IGeoConfig, IGeoData } from "../../../interfaces/GeoChart";
 import { TOP } from "../../visualizations/chart/legend/PositionTypes";
-import { isTwoDimensionsData } from "../../../helpers/executionResultHelper";
+import { getColorPaletteMapping } from "../geoChart/geoChartColor";
+import Legend, { ILegendProps } from "../../visualizations/chart/legend/Legend";
+
+export interface ISegmentByItemLegend {
```

Comment:
It could be `IPushpinCategoryLegendItem`

Reply:
fixed